backend/src/main/java/com/yagubogu/global/config/ClientConfig.java

```diff
@@ -0,0 +1,25 @@
+package com.yagubogu.global.config;
+
+import java.time.Duration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.client.SimpleClientHttpRequestFactory;
+import org.springframework.web.client.RestClient;
+
+@Configuration
+public class ClientConfig {
```

Comment:
C: RestClient가 많아지면 의미가 모호해질 것 같아요

```suggestion
public class KboClientConfig {
```

Reply:
한마디 보태자면, 저도 구글 서버에 인증을 보낼 때  RestClient를 사용했는데, restClient마다 타임아웃 설정도 다르고 baseUrl도 다르기 때문에 분리하는게 맞지 않을까 생각해요!